pydoctor/test/test_sphinx.py

```diff
@@ -402,6 +402,33 @@ def test_parseInventory_single_line(inv_reader_nolog: sphinx.SphinxInventory) ->
     assert {'some.attr': ('http://base.tld', 'some.html')} == result
 
 
+def test_parseInventory_spaces() -> None:
+    """
+    Check that columns that contain spaces are not split up.
```

Comment:
```suggestion
    When parsing an Sphinx inventory line, it will always return 5 values, even when some inventory column values contains spaces.
```

Reply:
I'll update the docstring.